cpp/include/graphar/util/filesystem.h

```diff
@@ -35,16 +35,19 @@
 namespace arrow {
 class Buffer;
 class Table;
-namespace fs {
+}// namespace arrow
```

Comment:
the correct format is 
```suggestion
} // namespace arrow
```